test/test_group.cpp

```diff
@@ -2190,6 +2190,100 @@ TEST(Group_CascadeNotify_Simple)
     CHECK(called);
 }
 
+/*
```

Comment:
Is it intentional, that these tests are commented out. If so, I think we need to know why? If not they should be enabled. As always, comments about what the tests are trying to expose and how would be nice.